index.dd

```diff
@@ -662,9 +662,19 @@ $(DIVC frontpage-orgs center,
     </div>
 )))
 
+)) $(COMMENT SECTION2 Why D?, DIVC whyd)
 
+$(DIVC call_to_donate,
+    <h1>D Language Foundation</h1>
```

Comment:
There shall only be one `h1` on a page (otherwise search engines will hate you).
In any case it's a bit too large, so how about `h2`?